test/functional/p2p_orphan_handling.py

```
@@ -0,0 +1,414 @@
+        peer.send_and_ping(msg_tx(tx))
+
+    @cleanup
+    def test_arrival_timing_orphan(self):
+        self.log.info("Test missing parents that arrive during delay are not requested")
+        node = self.nodes[0]
+        tx_parent_arrives = self.wallet.create_self_transfer()
+        tx_parent_doesnt_arrive = self.wallet.create_self_transfer()
+        # Fake orphan spends nonexistent outputs of the two parents
+        tx_fake_orphan = self.wallet.create_self_transfer_multi(utxos_to_spend=[
+            {"txid": tx_parent_doesnt_arrive["txid"], "vout": 10, "value": tx_parent_doesnt_arrive["new_utxo"]["value"]},
+            {"txid": tx_parent_arrives["txid"], "vout": 10, "value": tx_parent_arrives["new_utxo"]["value"]}
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)